db/storage.go

```diff
@@ -0,0 +1,94 @@
+package db
+
+import (
+    "database/sql"
+    _ "github.com/go-sql-driver/mysql"
+    "strconv"
+
+    "./../error"
```

Comment:
Use full path, `github.com/grubastik/flat-search/db`. On you local machine code should be located in `~/go/src/github.com/grubastik/flat-search`